packages/terra-status-view/src/terra-dev-site/test/status-view/StatusViewTestCommon.module.scss

```diff
@@ -7,5 +7,6 @@
   .status-view-wrapper-2 {
     border: 1px solid #000;
     width: 75px;
+    height: 500px;
```

Comment:
we have a lint error here that needs to be fixed https://travis-ci.com/cerner/terra-core/jobs/259205761#L314